### PR TITLE
ID:FPCO-12691; HOURS: 2; DONE: 90; separate frontend and backend code 

### PR DESCRIPTION
**Pull request by meetkoriya13**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,53 +4,26 @@
...
-    "@gofynd/nitrozen-vue": "0.0.33",
-    "axios": "^0.27.2",
     "body-parser": "^1.20.0",
-    "convict": "^6.2.3",
     "cookie-parser": "^1.4.6",
-    "core-js": "^3.25.2",
     "dotenv": "^10.0.0",
     "express": "^4.18.1",
     "fdk-client-javascript": "git+https://github.com/gofynd/fdk-client-javascript.git#0.1.25",
     "fdk-extension-javascript": "git+https://github.com/gofynd/fdk-extension-javascript.git#v0.5.3",
```
Comment (brijeshgajjarfynd) [inline package.json]: move it to latest SDK and client version

Comment (meetkoriya13) [reply package.json]: Updated

File: web/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "test": "jest --updateSnapshot --projects src --no-cache --detectOpenHandles --json --outputFile=./coverage/coverage.json"
+  },
+  "jest": {
+    "projects": [
+      "src/jest.config.js"
+    ]
+  },
+  "dependencies": {
+    "@gofynd/nitrozen-vue": "0.0.33",
+    "axios": "^0.27.2",
```
Comment (vivek-gofynd) [inline web/package.json]: Check if we can use browser's fetch instead axios here @meetkoriya13

File: web/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "groot_frontend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build --dest ../dist",
+    "lint": "vue-cli-service lint",
+    "test": "jest --updateSnapshot --projects src --no-cache --detectOpenHandles --json --outputFile=./coverage/coverage.json"
```
Comment (vivek-gofynd) [inline web/package.json]: remove test cases related dependencies @meetkoriya13

File: package.json [hunk 4]
```diff
@@ -4,53 +4,26 @@
...
     "dotenv": "^10.0.0",
     "express": "^4.18.1",
-    "fdk-client-javascript": "git+https://github.com/gofynd/fdk-client-javascript.git#0.1.25",
-    "fdk-extension-javascript": "git+https://github.com/gofynd/fdk-extension-javascript.git#v0.5.3",
-    "ioredis": "^5.2.2",
-    "vue": "^2.6.14",
-    "vue-router": "^3.5.3",
-    "vuex": "^3.6.2",
-    "window-or-global": "^1.0.1"
+    "@gofynd/fdk-client-javascript": "1.3.5",
```
Comment (vivek-gofynd) [inline package.json]: Check if we can remove this dependency @meetkoriya13

File: package.json [hunk 5]
```diff
@@ -4,53 +4,26 @@
...
-    "babel-polyfill": "6.26.0",
-    "@vue/cli-plugin-babel": "~5.0.8",
-    "@vue/cli-plugin-eslint": "~5.0.8",
-    "@vue/cli-plugin-router": "~5.0.8",
-    "@vue/cli-plugin-vuex": "~5.0.8",
-    "@vue/cli-service": "~5.0.8",
-    "@vue/test-utils": "1.2.1",
-    "babel-core": "7.0.0-bridge.0",
-    "@babel/eslint-parser": "^7.19.1",
     "eslint": "^7.32.0",
```
Comment (vivek-gofynd) [inline package.json]: Remove this @meetkoriya13

File: app/routes/v1.router.js [hunk 6]
```diff
@@ -1,15 +1,8 @@
...
 const router = express.Router();
 
-router.get('/test-api', (req, res, next) => {
-    res.json({
-        "hello": "hello"
-    });
-});
-
 // Get applications list
 router.get('/applications', async function view(req, res, next) {
```
Comment (vivek-gofynd) [inline app/routes/v1.router.js]: Move this to common server.js file

File: web/package.json (hunk 2 above)
Comment (vivek-gofynd) [inline web/package.json]: remove this dependency @meetkoriya13

**Review by vivek-gofynd — CHANGES_REQUESTED** [review]
Also delete healthz routes and dotenv dependency from backend.

File: web/package.json (hunk 2 above)
Comment (meetkoriya13) [reply web/package.json]: @vivek-gofynd  Replaced with fetch.
However, there was an interceptor for sending `x-company-id` to fetch session at the backend. as fetch does not provide functionality like an interceptor. Currently sending headers API level only.
Let me know if need to revert this change.

File: web/package.json (hunk 3 above)
Comment (meetkoriya13) [reply web/package.json]: Done

**Pull request closed without merging**